app/src/main/java/app/rive/runtime/example/AndroidPlayerActivity.kt

```diff
@@ -337,12 +337,14 @@ class AndroidPlayerActivity : AppCompatActivity() {
                     text = animation.stateMachine.name
                 }
                 text?.let { theText ->
-                    val textView = TextView(that)
-                    textView.text = "Play $theText"
-                    events.addView(textView, 0)
-                    playButtonMap[theText]?.background?.setTint(Color.GREEN)
-                    pauseButtonMap[theText]?.background?.setTint(Color.WHITE)
-                    stopButtonMap[theText]?.background?.setTint(Color.WHITE)
+                    runOnUiThread {
```

Comment:
This was crashing with the Swappy update because these `notifyPlay/Pause/etc.()` are now triggered from the cpp render thread.
Android doesn't let it happen though and it crashes, hence the need to force it to happen on the UI thread

Reply:
yeah, good point, we probably should